.github/workflows/test_upload_build_to_test_flight.yml

```diff
@@ -51,12 +52,13 @@ jobs:
         MATCH_REPO: ${{ secrets.MATCH_REPO }}
         API_KEY_ID: ${{ secrets.API_KEY_ID }}
         ISSUER_ID: ${{ secrets.ISSUER_ID }}
+        TEAM_ID: ${{ secrets.TEAM_ID }}
```

Comment:
I think we don't get it from secrets anymore, can we remove it?

Reply:
We still need it :D